app/src/main/java/com/sun/unsplash_02/screen/home/HomeFragment.kt

```diff
@@ -36,6 +39,9 @@ class HomeFragment : BaseFragment(), HomeContract.View, PhotoAdapter.ItemClickLi
     }
 
     override fun onInit() {
+        val activity = getBaseActivity() as MainActivity
```

Comment:
anh thấy bên dưới có sử dung lại biến activity nên em có thể khai báo 1 biến toàn cục activity để tái sử dụng lại cho gọn code